s3_bucket_block/main.tf

```
@@ -100,7 +100,7 @@ module "bucket_config" {
 # -- Outputs --
 output "buckets" {
   description = "Map of the bucket names:ids created from bucket_data."
-  value       = zipmap(
-      sort(keys(aws_s3_bucket.bucket)[*]),
-      sort(values(aws_s3_bucket.bucket)[*]["id"]))
+  value = zipmap(
+    sort(keys(aws_s3_bucket.bucket)[*]),
+  sort(values(aws_s3_bucket.bucket)[*]["id"]))
```

very curious how this got unindented :/

```suggestion
    sort(values(aws_s3_bucket.bucket)[*]["id"]))
```

this is what I'd expect